app/views/snps/show.html.erb

```diff
@@ -59,6 +63,22 @@
       We don't have enough users yet.
     <%end%>
   </div>
+
+  <div class="span3 columns">
+    <center><h6><a href="#" rel="tooltip" data-html="true" title="Phenotypes possibly associated with this SNP.">Phenotypes</a></h6></center>
+    <% if @snp.phenotypes.count %>
+      <table id="phenotype_table" class="table table-striped">
+        <% PhenotypeSnp.where(:snp_id => @snp.id).order(:score).reverse.each do |p| %>
```

Comment:
Maybe move this piece into the Phenotype-SNP model or SNPs-show-controller?

Like 

```@ordered_phenotype_snps  = PhenotypeSnp.where(:snp_id => @snp.id).order(:score).reverse```